docs/dvm-system-contract.md

```diff
@@ -28,4 +27,83 @@ The system contracts are deployed in various networks as follows.
 | Sha256        | 0x0000000000000000000000000000000000000002 |
 | Ripemd160     | 0x0000000000000000000000000000000000000003 |
 | Identity      | 0x0000000000000000000000000000000000000004 |
-| WithDraw      | 0x0000000000000000000000000000000000000015 |
\ No newline at end of file
+| WithDraw      | 0x0000000000000000000000000000000000000015 |
+
+> The Transfer contract is a superset of the Withdraw contract, and the Crab network's Withdraw contract will be upgraded to the Transfer contract in the future.
+
+## Contract Docs
+
+In order to be as compatible as possible with Ethereum and reduce the burden for application migration, the system contract currently consists of two types, the pre-compiled contracts compatible with Ethereum(ECRecover, Sha256, Ripemd160, Identity) and the Pangolin-specific pre-compiled contract Transfer.
```

Comment:
Pangolin-specific or Darwinia-specific?

Reply:
This doc mainly explains Pangolin or Crab network design. The Darwinia network has its own doc site. So, I think `Pangolin-specific` would be better here.